inventory/group_vars/all.yml

```diff
@@ -388,6 +388,18 @@ nginx_sites:
         try_files /maintenance.html =503;
       }
 
+      location /cable {
+        {{ nginx_valid_methods }}
+        proxy_pass http://rails;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-Proto https;
+        proxy_set_header X-Forwarded-Ssl on;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+      }
+
+
+
```

Comment:
Sorry to be picky, but I think we should avoid avoid multiple blank lines for consistent style.

Reply:
I agree.